functions/error-logger.js

```diff
@@ -13,9 +13,7 @@ function initErrorLogger() {
 }
 
 async function reportError(error) {
```

Comment:
Is this async function needed? I see you call `Sentry.flush` but we are not returning a Promise or using await